models/classes/Platform/Repository/Lti1p3RegistrationRepository.php

```diff
@@ -41,25 +41,26 @@
 class Lti1p3RegistrationRepository extends ConfigurableService implements RegistrationRepositoryInterface
 {
     private const OIDC_URL = ROOT_URL . 'taoLti/Security/oidc';
+    private const OAUTH_URL = 'http://tao-nginx/taoLti/Security/oauth';
     private const PLATFORM_ID = 'tao';
 
     public function find(string $identifier): ?RegistrationInterface
     {
-        $ltiProvider = $this->getLtiProviderService()
```

Comment:
Why you removed it from here, the identifier in this case is the LTI provider id...